### ISSUE TITLE
Proposal : Rename doc-install to doc-admin

### ISSUE THREAD

**Issue by ZeikoFr** [glpi-project/doc-install#55]
To be more focused on the purpose of each documentation I think we should rename this doc in doc-admin.

This change would make the 3 docs repository with the same name as they are in the project page documentation : 
![image](https://user-images.githubusercontent.com/54892453/108097260-2f816280-7082-11eb-896b-ac4feda0a3b1.png)

And it will allow everyone to add a lot of other important information about managing a GLPI server (it's not just install and update) that are currently only in the old documentation.

I think we should follow the path of the nextcloud documentation that as three distinct manual : 
![image](https://user-images.githubusercontent.com/54892453/108097631-aae31400-7082-11eb-89bf-3e3bcd72d474.png)

Any opinion about this ?

**Comment by cedric-anne** [comment]
Indeed, this documentation contains more than just installation/update instructions. Changing the repository URL should not annoy too many people, given the few contributions outside of GLPI core developers.
